pkg/k8s/apis/cilium.io/v2alpha1/bgp_node_types.go

```diff
@@ -188,20 +189,16 @@ type CiliumBGPNodePeerStatus struct {
 	// +kubebuilder:validation:Optional
 	Timers *CiliumBGPTimersState `json:"timers,omitempty"`
 
-	// Uptime is the time since the last peering session was established.
+	// EstablishedTime is the time when the peering session was established.
+	// It is represented in RFC3339 form and is in UTC.
 	//
 	// +kubebuilder:validation:Optional
-	Uptime *string `json:"uptime,omitempty"`
+	EstablishedTime *string `json:"establishedTime,omitempty"`
```

Comment:
Are there any considerations with respect to cilium upgrades here?

Reply:
The CiliumBGPNodeConfig resource is internal; users do not modify it. Status fields were never set, so there are no consumers. I believe changing the fields should be safe. 

I remember I needed to bump the CRD version number for the upgrade so that it would automatically pick the new CRD version and update it in the latest push.